tonic/src/status.rs

```diff
@@ -588,6 +584,13 @@ impl Status {
         self.add_header(response.headers_mut()).unwrap();
         response
     }
+
+    /// Header name for the `grpc-status` header
+    pub const GRPC_STATUS: HeaderName = HeaderName::from_static("grpc-status");
+    /// Header name for the `grpc-message` header
+    pub const GRPC_MESSAGE: HeaderName = HeaderName::from_static("grpc-message");
+    /// Header name for the `grpc-status-details-bin` header
+    pub const GRPC_STATUS_DETAILS: HeaderName = HeaderName::from_static("grpc-status-details-bin");
```

Comment:
Im kinda torn on exposing these publically,I know we need to use them in tonic-web but I wonder if we could just do doc(hidden)?

Reply:
`#[doc(hidden)]` seems fine to me, will change.